package.json

```diff
@@ -1527,7 +1527,6 @@
     "@types/pidusage": "^2.0.2",
     "@types/pixelmatch": "^5.2.4",
     "@types/pngjs": "^3.4.0",
-    "@types/prettier": "^2.3.2",
```

Comment:
Deprecated type package, types now bundled in `prettier`. See https://www.npmjs.com/package/@types/prettier